service/matching/handler.go

```diff
@@ -80,9 +80,19 @@ func NewHandler(
 		Resource:      resource,
 		config:        config,
 		metricsClient: resource.GetMetricsClient(),
-		rateLimiter: quotas.NewDynamicRateLimiter(func() float64 {
-			return float64(config.RPS())
-		}),
+		rateLimiter: quotas.NewMultiStageRateLimiter(
+			func() float64 {
+				return float64(config.RPS())
+			},
+			func(domain string) float64 {
```

Comment:
is the domain used

Reply:
Used after changed the config to be filtered by domain.